A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IRSInit.js

```diff
@@ -0,0 +1,128 @@
+            } else {
+                statusIRS3 = "IRS3 ALIGNING ON GPS"
+            }
+            IRS3GpsString = IRSAlignOnPos + "[color]green"
+        } else {
+            statusIRS3 = "IRS3 OFF"
+        }
+
+        mcdu.setTemplate([
+            ["IRS INIT"],
+            ["LAT" + larrowupdwn , rarrowupdwn + "LONG", "REFERENCE",],
+            ['{IrsInitFont}' + originAirportLat['deg'] + '°{IrsInitFontEnd}' + originAirportLat['min'] + '.' + Math.ceil(Number(originAirportLat['sec'] / 100)) + '[s-text]{IrsInitFont}' + originAirportLat['dir'] + "{IrsInitFontEnd} [color]blue", '{IrsInitFont}' + originAirportLon['deg'] + '°{IrsInitFontEnd}' + originAirportLon['min'] + '.' + Math.ceil(Number(originAirportLon['sec'] / 100)) + '[s-text]{IrsInitFont}' + originAirportLon['dir'] + "{IrsInitFontEnd} [color]blue", mcdu.flightPlanManager.getOrigin().ident + " [color]green"],
+            ["LAT", "LONG", "GPS POSITION"],
+            GPSPosAlign,
+            ["", "", statusIRS1],
+            ["", "", IRS1GpsString],
+            ["", "", statusIRS2],
+            ["", "", IRS2GpsString],
+            ["", "", statusIRS3],
+            ["", "", IRS3GpsString],
+            [],
+            ["<RETURN", alignMsg]
+        ])
+        
+        // IRS Font is different we loop over keywords to set correct font since at the moment we cannot adapt FMCMainDisplay.js to allow for extra keywords
+        mcdu._lineElements.forEach(function (ele) {
+            ele.forEach(function (el) {
+                if (el != null) {
+                let newHtml = el;
+                if (newHtml != null) {
+                    newHtml = newHtml.innerHTML.replace(/{IrsInitFont}/g, '<span class=\'irs-text\'>');
+                    newHtml = newHtml.replace(/{IrsInitFontEnd}/g, '</span>');
+                    el.innerHTML = newHtml;
+                }
+            }})
+        });
+
+        mcdu.onLeftInput[0] = () => {
+            lon = false
+        };
+
+        mcdu.onRightInput[0] = () => {
+            lon = true
+        };
+
+        mcdu.onLeftInput[5] = () => {
+            CDUInitPage.ShowPage1(mcdu);
+        };
+
+        mcdu.onRightInput[5] = () => {
+            if (!SimVar.GetSimVarValue("A32NX_ADIRS_PFD_ALIGNED_FIRST", "Bool") && !SimVar.GetSimVarValue("L:A320_Neo_ADIRS_IN_ALIGN", "Bool")) {
+                SimVar.SetSimVarValue("L:A320_Neo_ADIRS_IN_ALIGN", "Bool", 1);
+            } else {
+                // Ref alignment not currently implemented
+            }
+        };
+
+
+        // This page auto-refreshes based on source material. Function will stop auto-refreshing when page has been changed.
+        autoRefresh().then()
+        async function autoRefresh() {
+            await new Promise(r => setTimeout(r, 2000));
+            if (mcdu.getTitle() == 'IRS INIT') {
+                CDUIRSInit.ShowPage(mcdu, lon = lon)
+            } else {
+                return
+            }
+        }
```

Comment:
I believe this is equivalent and reduces some complexity that isn't needed:
```suggestion
        autoRefresh();
        function autoRefresh() {
            setTimeout(() => {
                if (mcdu.getTitle() == 'IRS INIT') {
                    CDUIRSInit.ShowPage(mcdu, lon = lon);
                }
            }, 2000);
        }
```

Reply:
I agree, has been changed accordingly.